awsesproxy/trivyignore

```diff
@@ -1,3 +1,4 @@
 # Upstream awsesproxy images include several CVEs.
 # We should try to remove these each time we update.
 CVE-2021-44716
```

Comment:
is this CVE still showing up in the scan?

Reply:
this is golang.org/x/net module